doc/benchmark_workflow/config_benchopt.rst

```diff
@@ -22,19 +22,19 @@ Config File Location
 For the global configuration file, the resolution order is the following:
 
 1. The environment variable ``BENCHOPT_CONFIG`` is set to an existing file,
-2. A file ``benchopt.ini`` in the current directory,
-3. The default file is ``$HOME/.config/benchopt.ini``.
+2. A file ``benchopt.yml`` in the current directory,
+3. The default file is ``$HOME/.config/benchopt.yml``.
 
-For benchmark configuration files, they are usually located in the benchmark folder, and named ``benchopt.ini``. If it does not exist, the default is to use the global config file.
+For benchmark configuration files, they are usually located in the benchmark folder, and named ``benchopt.yml``. If it does not exist, the default is to use the global config file.
 
 .. _config_structure:
 
 Config File Structure
 ---------------------
 
-The config files for benchopt follow the Microsoft Windows INI files structure. The global setting are grouped in a ``[benchopt]`` section:
+The config files for benchopt follow the Microsoft Windows yml files structure. The global setting are grouped in a ``[benchopt]`` section:
 
-.. code-block:: ini
+.. code-block:: yml
 
     [benchopt]
```

Comment:
this is still .ini code no?

Reply:
yes, updated! thanks for pointing this out